pkgs/development/python-modules/pytest-responses/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, python3
+, fetchFromGitHub
+,
+}:
```

Comment:
```suggestion
{
  lib,
  buildPythonPackage,
  fetchFromGitHub,
  pytest,
  pytestCheckHook,
  responses,
}:
```

Reply:
If you declare the python packages necessary as part of the function inputs, you don't need to use `with python3.pkgs` everywhere.